AWSAuthSDK/Sources/AWSMobileClient/AWSMobileClientExtensions.swift

```diff
@@ -505,24 +505,73 @@ extension AWSMobileClient {
         }
     }
     
+    private func hostedUILegacySignOut(completionHandler: @escaping ((Error?) -> Void)) {
```

Comment:
We've favored putting the more specialized methods below the "switch" conditional that delegates to the specialized implementation, so it's easier to read through the flow. So if `switchingMethod` delegates to `methodA` in one case and `methodB` in the other case, declare the methods in the order "switchingMethod", "methodA", "methodB".

In this case, this file is so large that the benefit would largely be meaningless, but it would still be nice to keep the habit going.

Reply:
Updated